app/javascript/components/physical-storage-form/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import MiqFormRenderer from '../../forms/data-driven-form';
```

Comment:
Please import this from `@@ddf` but treat is as an internal import.

Reply:
^

Reply:
@skateman are you sure about "import MiqFormRenderer from '../../forms/data-driven-form';" ?
in all other forms, MiqFormRenderer is imported this way.

Reply:
Yeah, it's basically a shortcut pointing to the same file. I guess all other forms should be changed too, I'll take care of that...